code/game/objects/structures/inflatable.dm

```diff
@@ -243,7 +243,7 @@
 	verbs -= /obj/structure/inflatable/verb/hand_deflate
 	deflate()
 
-/obj/structure/inflatable/proc/update_nearby_tiles(var/turf/T)
+/proc/update_nearby_tiles(var/turf/T)
```

Comment:
very questionable. what's the intent here? `get_turf(src)` is called by this proc and `src` is always null for global procs

Reply:
It was just an oversight, it was moved to atom-level a few seconds before you made this comment.